### PR TITLE
Opencast 2950 add pop up when remove captions clicked on lti manage

### PR DESCRIPTION
**Pull request by npmali**
### Your pull request should…

* [ ] have a concise title
* [ ] [close an accompanying issue](https://help.github.com/en/articles/closing-issues-using-keywords) if one exists
* [ ] be against the correct branch (features can only go into develop)
* [ ] include migration scripts and documentation, if appropriate
* [ ] pass automated testing
* [ ] have a clean commit history
* [ ] have proper commit messages (title and body) for all commits
* [ ] have appropriate tags applied

### CODE REVIEW

**Review by NgoniChoga — APPROVED** [review]

File: modules/lti/src/main/resources/tools/manage/js/lti.manage.js [hunk 1]
```diff
@@ -2596,22 +2596,33 @@ $(document).ready(function() {
    });
    $('#editPublishedModal').on('click', '#rmGoogleCaptions, #rmNibityCaptions, #rmUploadedCaptions', function(e) {
```
Comment (TurRil) [inline modules/lti/src/main/resources/tools/manage/js/lti.manage.js:2597]: Check your indentation for this chunk of code - seems to be inconsistent with the rest of the file.

File: modules/lti/src/main/resources/tools/manage/js/lti.manage.js [hunk 2]
```diff
@@ -2596,22 +2596,33 @@ $(document).ready(function() {
...
-        captionsText = "",
+        modalTitle = "",
         captionsProvider = $(this).attr('data-provider');
 
+    $('#removeCaptionsModal #eventDetails').attr('data-event', eventId);
+    $('#removeCaptionsModal #eventDetails').attr('data-provider', captionsProvider);
+
     if(captionsProvider === "googleTranscript") {
-       captionsText = "Automated Google captions will be removed from this video.";
+       modalTitle = "Remove Automated captions";
```
Comment (TurRil) [inline modules/lti/src/main/resources/tools/manage/js/lti.manage.js:2606]: change to "Remove Automated Captions"  to make it consistent with labels on "Download Captions" block

File: modules/lti/src/main/resources/tools/manage/js/lti.manage.js [hunk 3]
```diff
@@ -2596,22 +2596,33 @@ $(document).ready(function() {
...
     }
     if(captionsProvider === "nibityTranscript") {
-       captionsText = "Way with Words captions will be removed from this video.";
+       modalTitle = "Remove Way with words captions";
```
Comment (TurRil) [inline modules/lti/src/main/resources/tools/manage/js/lti.manage.js:2609]: change to "Remove Way with Words captions"  to make it consistent with labels on "Download Captions" block

File: modules/lti/src/main/resources/tools/manage/js/lti.manage.js [hunk 4]
```diff
@@ -2596,22 +2596,33 @@ $(document).ready(function() {
...
     }
     if(captionsProvider === "uploadedTranscript") {
-       captionsText = "Uploaded captions will be removed from this video.";
+       modalTitle = "Remove uploaded captions";
```
Comment (TurRil) [inline modules/lti/src/main/resources/tools/manage/js/lti.manage.js:2612]: change to "Remove Uploaded Captions"  to make it consistent with labels on "Download Captions" block

File: modules/lti/src/main/resources/tools/manage/index.html [hunk 5]
```diff
@@ -572,6 +572,36 @@ <h4 class="modal-title" id="rtModal">Transcript Request</h4>
...
+                  <p>Removing captions cannot be undone. If you want better captions for this video, you can edit
+                      the existing captions, or upload new captions or request better captions.</p>
+                  <br/>
+                  <p>If you remove automated captions without uploading new captions, you will not be able to request better captions.
+                      Please upload new captions before removing automated captions.</p>
+                  <br/>
+                  <p>If you are sure about removing this transcript click "Yes, Remove Transcript". If you are not sure click "Cancel" and try
+                      using the other available options.</p>
+
+                  <input type="hidden" id="eventDetails"/>
```
[the remark this answers is not in the document]
Comment (TurRil) [reply modules/lti/src/main/resources/tools/manage/index.html:596]: ignore this - it's a hidden field that you populate

**Review by TurRil — CHANGES_REQUESTED** [review]

**Pull request merged**